src/Utilities/FlowAnalysis/Analysis/ValueContentAnalysis/ValueContentAbstractValue.cs

```diff
@@ -120,9 +120,24 @@ private static ValueContentAbstractValue Create(ImmutableHashSet<object> literal
             return new ValueContentAbstractValue(literalValues, nonLiteralState);
         }
 
-        public static bool IsSupportedType(ITypeSymbol type)
+        public static bool IsSupportedType(ITypeSymbol type, out ITypeSymbol valueTypeSymbol)
```

Comment:
@mavasani I haven't deeply looked into the ValueContentAnalysis changes, so please yell if I'm doing something silly.